server/src/main/java/org/opensearch/bootstrap/BootstrapChecks.java

```diff
@@ -222,11 +223,32 @@ static List<BootstrapCheck> checks() {
         checks.add(new OnErrorCheck());
         checks.add(new OnOutOfMemoryErrorCheck());
         checks.add(new EarlyAccessCheck());
+        checks.add(new JvmVersionCheck());
         checks.add(new AllPermissionCheck());
         checks.add(new DiscoveryConfiguredCheck());
         return Collections.unmodifiableList(checks);
     }
 
+    static class JvmVersionCheck implements BootstrapCheck {
```

Comment:
Should we call this JavaVersionCheck? Since really we're checking which specific version of *Java* is being used.

Reply:
Sure, renamed, thanks!